server.js

```diff
@@ -1,9 +1,120 @@
 import express from 'express';
+import path from 'path';
+import passport from 'passport';
+import passport_facebook from 'passport-facebook';
+import session from 'express-session';
+import cookieParser from 'cookie-parser';
+
 
 const server = express();
+const FacebookStrategy = passport_facebook.Strategy;
+
+server.use(cookieParser());
+server.use(session({ secret: 'keyboard cat' }));
+// Initialize Passport!  Also use passport.session() middleware, to support
+// persistent login sessions (recommended).
+server.use(passport.initialize());
+server.use(passport.session());
+
+
+
+//TODO may want these secure somewhere?
+var FACEBOOK_APP_ID = "819934004772176";
+var FACEBOOK_APP_SECRET = "90ff6c2c0231f2b185acb5ecdc74dfae";
+
+
+// Passport session setup.
+//   To support persistent login sessions, Passport needs to be able to
+//   serialize users into and deserialize users out of the session.  Typically,
+//   this will be as simple as storing the user ID when serializing, and finding
+//   the user by ID when deserializing.  However, since this example does not
+//   have a database of user records, the complete Facebook profile is serialized
+//   and deserialized.
+passport.serializeUser(function (user, done) {
+    console.log('serializing user');
+    done(null, user);
+});
+
+passport.deserializeUser(function (obj, done) {
+    console.log('deserializing user');
+    done(null, obj);
+});
+
+// Use the FacebookStrategy within Passport.
+//   Strategies in Passport require a `verify` function, which accept
+//   credentials (in this case, an accessToken, refreshToken, and Facebook
+//   profile), and invoke a callback with a user object.
+passport.use(new FacebookStrategy({
+    clientID: FACEBOOK_APP_ID,
+    clientSecret: FACEBOOK_APP_SECRET,
+    callbackURL: "http://localhost:3000/auth/facebook/callback"
```

Comment:
Will this change when it's hosted on a server?  I can see it being fine, it just feels weird to read `localhost`